components/tree/style/mixin.less

```diff
@@ -1,4 +1,4 @@
-@import '../../style/mixins/index';
+﻿@import '../../style/mixins/index';
```

Comment:
Please fix the encoding.

Reply:
The file encoding is changed from utf-8 to utf-8 with bom，so the changing chars is �ｿ